micromamba/tests/test_env.py

```diff
@@ -70,31 +70,47 @@ def export_env():
     return env_name
 
 
+@pytest.fixture(scope="module")
+def empty_env():
+    env_name = "env-empty"
+    helpers.create("-n", env_name)
+    return env_name
+
+
+@pytest.mark.parametrize("json_flag", [None, "--json"])
+def test_empty_env_export(json_flag, empty_env):
+    flags = filter(None, [json_flag])
+    output = helpers.run_env("export", "-n", empty_env, *flags)
+
+    # json is already parsed
+    ret = output if json_flag else yaml.safe_load(output)
+    assert ret["name"] == empty_env
+    assert "env-empty" in ret["prefix"]
+    assert not ret["channels"]
+
+
 @pytest.mark.parametrize("channel_subdir_flag", [None, "--channel-subdir"])
 @pytest.mark.parametrize("md5_flag", [None, "--md5", "--no-md5"])
 @pytest.mark.parametrize("explicit_flag", [None, "--explicit"])
 @pytest.mark.parametrize("no_build_flag", [None, "--no-build", "--no-builds"])
 @pytest.mark.parametrize("json_flag", [None, "--json"])
 def test_env_export(
-    export_env, json_flag, no_build_flag, explicit_flag, md5_flag, channel_subdir_flag
+    channel_subdir_flag, md5_flag, explicit_flag, no_build_flag, json_flag, export_env
```

Comment:
This is the order it is mentioned above, it's better to have the same order everywhere